server/routerlicious/packages/routerlicious-base/package.json

```diff
@@ -89,9 +89,9 @@
 		"@fluidframework/build-common": "^1.1.0",
 		"@fluidframework/build-tools": "^0.16.1",
 		"@fluidframework/eslint-config-fluid": "^2.0.0",
-		"@fluidframework/server-local-server": "^0.1041.1000",
+		"@fluidframework/server-local-server": "workspace:*",
 		"@fluidframework/server-routerlicious-base-previous": "npm:@fluidframework/server-routerlicious-base@0.1040.1000",
```

Comment:
Shouldn't this have changed to `0.1041.1000` after rebuilding type tests?

Reply:
No, because 0.1040.1000 is the latest released version. The release group was bumped to the next major after the last release, which at the time was 1041. But 1041 has never been released.